todo_app/test_seleniumtests.py

```diff
@@ -0,0 +1,140 @@
+import os, pytest, requests
+from dotenv import load_dotenv, find_dotenv
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.ui import Select
+from threading import Thread
+from todo_app import app
+
+def create_board(board_name):
+    url = 'https://api.trello.com/1/boards/'
+    api_params = params({ 'name': board_name })
+    response = requests.post(url, api_params).json()
+    board_id = response['id']
+    rename_lists(board_id)
```

Comment:
Nice work, most people back out and change their lists but keeping them as we want them is nicer! 🙂